post_installation_configuration/configuring-multi-arch-compute-machines/creating-multi-arch-compute-nodes-azure.adoc

```diff
@@ -6,14 +6,18 @@ include::_attributes/common-attributes.adoc[]
 
 toc::[]
 
-To deploy an Azure cluster with multi-architecture compute machines, you must first create a single-architecture Azure installer-provisioned cluster that uses the multi-architecture installer binary. For more information on Azure installations, see xref:../../installing/installing_azure/installing-azure-customizations.adoc#installing-azure-customizations[Installing a cluster on Azure with customizations]. You can then add an ARM64 compute machine set to your cluster to create a cluster with multi-architecture compute machines.
+To deploy an Azure cluster with multi-architecture compute machines, you must first create a single-architecture Azure installer-provisioned cluster that uses the multi-architecture installer binary. For more information on Azure installations, see xref:../../installing/installing_azure/installing-azure-customizations.adoc#installing-azure-customizations[Installing a cluster on Azure with customizations]. 
```

Comment:
```suggestion
To deploy an Azure cluster with multi-architecture compute nodes, you must first create a single-architecture Azure installer-provisioned cluster that uses the multi-architecture installer binary. For more information on Azure installations, see xref:../../installing/installing_azure/installing-azure-customizations.adoc#installing-azure-customizations[Installing a cluster on Azure with customizations]. 
```

We use to call this scenario `Multi-architcture compute nodes clusters`. Leaving this decision up to you

Reply:
This is a really good point. I also notice that we have used "machines" interchangeably with "nodes" in many places. I'd like to take a closer look at this and work on maintaining a consistent terminology across the docs.
For now, I'd leave this as is. But I pick this up on priority in the future.